pyproject.toml

```diff
@@ -54,6 +54,7 @@ identify-demo = "examples.identify.identify:main"
 identify-push-demo = "examples.identify_push.identify_push_demo:run_main"
 identify-push-listener-dialer-demo = "examples.identify_push.identify_push_listener_dialer:main"
 pubsub-demo = "examples.pubsub.pubsub:main"
+kademlia-demo = "examples.kademlia.kademlia:main"
```

Comment:
Since the kademlia demo is more complicated, I think you can remove it from the `scripts` list here and in `contributing.rst` and just have it be a more manual process.

If you want to update the demo it to use the script syntax instead, that's fine too.